### ISSUE TITLE
Crash on Startup

### ISSUE THREAD

**Issue by jlmcmchl** [PleasingFungus/Silicon-Zeroes#95]
ERROR: Creating cursor from art/cursors/pen_x.png: SDL ERROR (CreateIconIncorrect(): The parameter is incorect)

VERSION 1.1.0.1-g7e2312dc

**Comment by PleasingFungus** [comment]
Same as #25, which I thought I'd fixed... :(

**Comment by jlmcmchl** [comment]
For what it's worth, reinstalling the game fixed the issue, while verifying game cache did not.

**Comment by PleasingFungus** [comment]
Oh, that's interesting - it happened consistently?

**Comment by jlmcmchl** [comment]
Yep, every time I tried to open Silicon Zeroes it would crash with that error. When I reinstalled it, it ran just fine.

**Comment by PleasingFungus** [comment]
Ah, that is different!

What OS are you running?

**Comment by jlmcmchl** [comment]
Windows 10.

**Comment by PleasingFungus** [comment]
Gotcha. I don't have any immediate clues, but I'll keep thinking about it - thanks for reporting!
